agent/src/api/v1/jobs.rs

```diff
@@ -26,13 +26,15 @@ pub fn mount_proxy(router: ServalRouter) -> ServalRouter {
 async fn proxy(State(state): State<AppState>, mut request: Request<Body>) -> impl IntoResponse {
     let path = request.uri().path();
     log::info!("relaying a job runner request; path={path}");
+    metrics::increment_counter!("proxy:{path}");
 
     if let Ok(resp) =
         super::proxy::relay_request(&mut request, SERVAL_SERVICE_RUNNER, &state.instance_id).await
     {
         resp
     } else {
         // Welp, not much we can do
+        metrics::increment_counter!("proxy:{path}:error");
```

Comment:
Thoughts on changing this to `proxy:error:` so wildcard searches will be easier to write?